xbmc/filesystem/HTTPDirectory.cpp

```diff
@@ -98,8 +98,17 @@ bool CHTTPDirectory::GetDirectory(const CURL& url, CFileItemList &items)
       std::string strLinkBase = strLink;
       std::string strLinkOptions;
 
+      //convert to URL encode
+      size_t pos = strLinkBase.find_first_of("+;");
+      while (pos != std::string::npos) {
```

Comment:
move curly bracket into a new line

Reply:
Done 👍

Reply:
please squash the commits, for a clean commit history

Reply:
Done 👍